### DISCUSSION TITLE
Disc usage for scanned plugins

### DISCUSSION THREAD

**Discussion by Yorshoff** [apohl79/audiogridder#390] [Help] [answered]
Hi, guys.

I've installed AudioGridder Server on my Mojave (10.14.2) machine and during the initial plugin scan AudioGridder used enormous disc space (several tens of gigabytes). After scan I closed the program and during the next launch Server rescanned plugins again and used another huge portion of disk space. I've tried to uninstall AudioGridder Server, but after that disc space is still occupied with created data. 

Could you please help me to find the folder where AudioGridder store all that data, so I could delete it. 

Thanks

**Comment by apohl79** [comment, marked as answer]
Please check the /cores directory, even though AG does not enable core dumps for plugin scans, it might be enabled and if plugins crash AG during the scan, the cores might be eating up your disk space. You can also check the log file folders under ~/Library/Logs/AudioGridder/. But each scan is producing a very small text file only.

**Reply by Yorshoff** [reply]
Thanks! AudioGridder Log folder is only 9.6 MB on disk, so that's not the case looks like. "Please check the /cores directory" - can you, please, explain how to do so?

**Reply by apohl79** [reply]
Either open a terminal and enter:

ls -l /cores

or open finder and hit cmd+shift+g and enter /cores and press "go". This will open the /cores folder.

**Reply by Yorshoff** [reply]
Yes! In /cores folder I see a bunch of files created today during the time when I tried to use AG. Here is a screenshot. Is it safe to delete these files? How can I prevent my system from creating these files?
![Screen Shot 2021-02-19 at 17 07 05](https://user-images.githubusercontent.com/79320081/108522113-220fe680-72d5-11eb-962f-818f6be6d565.png)

**Reply by apohl79** [reply]
Yes, delete them

**Reply by Yorshoff** [reply]
> Yes, delete them

How can I prevent my system from creating these files?

**Reply by apohl79** [reply]
You can remove write permission from the directory:

Open a terminal (Applications/Utilities/Terminal.app).

Enter: sudo chmod og-w /cores

**Reply by Yorshoff** [reply]
Thanks! Is it a safe operation?

**Reply by apohl79** [reply]
Yes, the AG installer has enabled write permission for all users in the first place to be able to write corefiles, as that is usually needed to analyze problems. Now it will not cause problems on your machine to take that away again, as macOS does usually not enable core file creation.

**Reply by Yorshoff** [reply]
ok, I see now! Thanks a lot for your help!
